### PR TITLE
Small logging improvements (fixes #1037)

### LINKED ISSUE #1037: Store version string in logs
It will help us in debugging, in particular `letsencrypt-auto` users that run versioned PyPI releases. That's probably good candidate for `logging.INFO` level.

### PR DESCRIPTION
**Pull request by kuba**

### CODE REVIEW

**Comment by kuba** [comment]
Forgot to tag it in commits, but it fixes https://github.com/letsencrypt/letsencrypt/issues/1037.

**Comment by bmw** [comment]
LGTM. Merging when Travis has a prettier color.

File: acme/acme/standalone.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def __init__(self):
 
     def serve_forever2(self):
         """Serve forever, until other thread calls `shutdown2`."""
+        logger.debug("Starting server at %s:%d...", *self.socket.getsockname())
```
Comment (pde) [inline acme/acme/standalone.py]: Will this work if the return value is an IPv6 4-tupple?

Comment (kuba) [inline acme/acme/standalone.py]: It will not work. Good catch!

**Comment by kuba** [comment]
Fixed `getsockname` and Travis is now green...

**Comment by bmw** [comment]
Green is a prettier color. LGTM.

**Pull request merged**